apps/judicial-system/backend/migrations/20241022142128-update-defendant.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+module.exports = {
+  up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.transaction((transaction) =>
+      queryInterface.addColumn(
+        'defendant',
+        'verdict_appeal_date',
```

Comment:
I agree, the event log would be a better place to store this date. Adding `defendant_id` to that table would bring it's own set of challenges because then we'd be required to add logic everywhere we use that data, i.e. `if verdict_appeal_date then get by defendant id, otherwise get by case id`. I'll create a ticket in Asana so that we don't forget to talk about this.